### DISCUSSION TITLE
Is there a way to unstale only if updated by author?

### DISCUSSION THREAD

**Discussion by dundargoc** [actions/stale#887] [Q&A] [answered]
I couldn't find any option to do this, but asking just to make sure I didn't miss it. Like the title says, I'm only interested when am issue or PR was updated by the author only and was wondering if there was any way to do this?

**Comment by dundargoc** [comment, marked as answer]
I created my own workflow to do this for me. We can close this.

**Reply by stdedos** [reply]
Would you mind giving us an example of what you did? 🙏

**Reply by dundargoc** [reply]
Of course. I've written down the solution here: https://github.com/actions/stale/issues/719#issuecomment-1858310205
